_Week5/group4/program.do

```diff
@@ -0,0 +1,27 @@
+cap prog drop graphsum
+prog define graphsum
+	syntax anything
+	local sysdata: word 1 of `anything'
+	local var1: word 2 of `anything'
+	local var2: word 3 of `anything'
+	sysuse `sysdata', clear
+	twoway (scatter `var1' `var2') (lfit `var1' `var2')
+	graph save `sysdata'_scatter_`var1'_`var2'.gph, replace
+	cap matrix drop `sysdata'_summary
+	forval i=1/2{
+		qui sum `var`i''
+		mat `sysdata'_summary = nullmat(`sysdata'_summary)\[r(N), r(mean), r(Var), r(sd), r(min), r(max)]

```

Comment:
It would be worth explaining the code in comments for the user to understand. In particular, you may consider adding the fact that you use Mata, which is a separate language for matrices. The documentation on the language is unfortunately sparse, last I checked.